### PR TITLE
feat(linters):add linters config

### PR DESCRIPTION
**Pull request by GhislainMitahi**
### **WHAT I DID SO FAR**

Created a new app. Give it a meaningful name Railsblog
The project has Postgres database set up.
Built my schema.
Created and ran the necessary migration files.
Table and column names match the ERD diagram.
Here is migrating created
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,160 @@
-# README
+<a name="readme-top">Railsblog</a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <img src="./public/logo.svg" alt="logo" width="256"  height="auto" />
+  <br/>
```
Comment (GzimAsani) [inline README.md:5]: - [ ] The logo is broken kindly fix it

File: README.md [hunk 2]
```diff
@@ -1,24 +1,160 @@
...
+
+- Hat tip to anyone whose code was used.
+- Thanks for all the curated content that was provided to us.
+- Thanks to my learning and coding partners for all their support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (GzimAsani) [inline README.md:158]: - [x] Please add a MIT link, currently it leads to a 404 page

File: db/migrate/20230504130953_create_user.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```
Comment (GzimAsani) [inline db/migrate/20230504130953_create_user.rb]: - [ ] Make bio  t.text which will be bigger place for a longer messages

File: db/migrate/20230504132145_create_comment.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (GzimAsani) [inline db/migrate/20230504132145_create_comment.rb]: - [ ] Please here make a reference to the author similar what you did to the post migration
`t.references :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20230504132212_create_like.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```
Comment (GzimAsani) [inline db/migrate/20230504132212_create_like.rb]: - [x] Please here make a reference to the author similar what you did to the post migration
`t.references :author, foreign_key: { to_table: 'users' }`

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @GhislainMitahi              :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the migrations :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down:  

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (GhislainMitahi) [reply README.md:158]: okay good, thank you

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @GhislainMitahi , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**